CHANGELOG.md

```diff
@@ -16,6 +16,40 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## Unreleased
 
+## [3.0.0](https://github.com/torchbox/tbxforms/releases/tag/v3.0.0)
+
+### Developer
+
+-   Guidance on how to update `govuk-frontend` [#80]
+
+### Pinned packages updated
+
+-   `govuk-frontend` from 3.13.0 (possibly) to 5.4.1 [#80]
+
+### Removed
+
+Several CSS classes/mixins have been removed/renamed in this release, some of
+which are vendor changes in `govuk-frontend`.
+
+Unfortunately, the initial release of tbxforms customised the styles from
+`govuk-frontend` without documentation - including which version was used as a
+base. As a result, it's hard to distinguish between variables/mixins we
+added which have now been removed and those which GDS have removed over time.
+
+However, variables which have now been removed from our \_variables.scss are:
+
+-   `$tbxforms-border-colour-conditional` (no longer configurable; now using GDS' `mid-grey` (#b1b4b6)) [#80]
```

Comment:
If these values aren't configurable any more, what is the correct way to override the GDS defaults?

Reply:
I cannot for the life of me remember why I chose to hardcode this instead of reusing the `$tbxforms-border-colour` variable, but I assume this is a mistake!

Resolved in https://github.com/torchbox/tbxforms/pull/80/commits/f5fc73043412f0de42617a2a34ecf99d26491030

Thanks 🙌